polars/polars-core/src/fmt.rs

```diff
@@ -382,8 +382,48 @@ impl Display for DataFrame {
                 constraints.push(tbl_lower_bounds(l));
             }
             let mut table = Table::new();
-            let preset = if std::env::var("POLARS_FMT_NO_UTF8").is_ok() {
-                ASCII_FULL
+            // let preset = if std::env::var("POLARS_FMT_NO_UTF8").is_ok() {
+            //     if std::env::var("POLARS_FMT_NO_SEPARATOR_TABLES").is_ok() {
+            //         NOTHING
+            //     } else {
+            //         ASCII_FULL
+            //     }
+            // } else {
+            //     if std::env::var("POLARS_FMT_NO_SEPARATOR_TABLES").is_ok() {
+            //         NOTHING
+            //     } else {
+            //         UTF8_FULL
+            //     }
+            // };
+
+            let preset = if std::env::var("POLARS_FMT_TABLE_FORMATTING").is_ok() {
+                let str_preset =
+                    std::env::var("POLARS_FMT_TABLE_FORMATTING").unwrap_or("none".to_string());
+                if str_preset == "ASCII_FULL" {
```

Comment:
Im not a Rust programmer, but this could probably be rewritten in a much conciser way by using [match](https://doc.rust-lang.org/rust-by-example/flow_control/match.html) ?

```rust
match str_preset {
    "ASCII_FULL" => ASCII_FULL,
    "ASCII_NO_BORDERS" => ASCII_NO_BORDERS,
    ...
}